### DISCUSSION TITLE
AMD HIP SDK for Windows has been officially released

### DISCUSSION THREAD

**Discussion by Oggom** [vladmandic/sdnext#1852] [General]
It has finally happened:

https://www.amd.com/en/developer/rocm-hub/hip-sdk.html

**Comment by yamfun** [comment]
AMD user here, it seems we still need to wait at least for a port of MIOpen, and then wait for PyTorch to support, and then wait for the various SD UIs to support it
https://github.com/RadeonOpenCompute/ROCm/discussions/2347

Seems so distant that perhaps waiting for the price drop of 4060 TI 16GB and switch to green team will be faster
